host_applications/linux/apps/raspicam/RaspiHelpers.c

```diff
@@ -280,3 +280,17 @@ int mmal_status_to_int(MMAL_STATUS_T status)
       return 1;
    }
 }
+
+
+uint64_t get_microseconds64()
+{
+   struct timespec spec;
+   uint64_t ms;
```

Comment:
As ``ms`` usually means milliseconds, this made the following code look very wrong to me.
I'd prefer ``us`` or ``microsecs`` or ``microseconds`` to avoid the confusion.

Reply:
Absolutely right - a C&P from code that worked in milliseconds. Will fix up tomorrow.